core/src/executor/evaluate/stateless.rs

```diff
@@ -390,5 +390,10 @@ fn evaluate_function<'a>(context: &Context<'_>, func: &'a Function) -> Result<Ev
 
             f::concat_ws(name, separator, exprs)
         }
+        Function::Append { expr, value } => {
+            let expr = eval(expr)?;
+            let value = eval(value)?;
+            f::append(expr, value)
+        }
```

Comment:
to cover this, for now.. the most simple way would be this below,
```sql
CREATE TABLE Foo (
    id INTEGER,
    values LIST DEFAULT '[1,2,3]'
);
```
like this above, using `DEFAULT` can reach this lines.
Could you add this kind of test case to `test-suite/ function/append.rs`?